common/changes/@microsoft/rush/test-pnpm-shrinkwrap-file_2023-06-23-08-39.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Add test for PnpmShrinkwrapFile#isWorkspaceProjectModifiedAsync",
```

```suggestion
      "comment": "",
```

This isn't relevant to consumers.